pyclient/README.md

```diff
@@ -62,6 +62,203 @@ $ pip3 install dist/pydeephaven-0.6.0-py3-none-any.whl
 
 ```
 
+## Initialize
+
+The `Session` class is your connection to Deephaven. This is what allows your Python code to interact with Deephaven.
+
+```
+from pydeephaven import Session
+
+#session = Session(host=”envoy”) # Use this if you’re running the Python script in docker-compose with the Deephaven default settings
+session = Session() # Use this if you’re running the python script locally with with Deephaven default settings
+```
+
+## Ticking table
+
+The `Session` class has many methods that create tables. This example creates a ticking time table and binds it to Deephaven.
+
+```
+from pydeephaven import Session
+
+session = Session()
+
+table = session.time_table(period=1000000000).update(formulas=["Col1 = i % 2"])
+session.bind_table(name="MyTable", table=table)
+```
+
+This is the general flow of how the Python client interacts with Deephaven. You create a table (new or existing), execute some operations on it, and then bind it to Deephaven.
+
+If you don’t bind the table, it will not be updated in Deephaven.
```

Comment:
It isn't clear to me.  Find out what it does so that we can document properly.

Reply:
```
        """ Bind a table to the given name on the server so that it can be referenced by that name.
        Args:
            name (str): name for the table
            table (Table): a Table object
        Raises:
            DHError
        """
```

This is what the pydoc comment says. I'll update the README to reflect this